MovingBlocks/Terasology
# fix(PathManager): allow paths to be symlinks

Useful for directories like cachedModules.

### How to test

In your Terasology home directory, replace `cachedModules` with a symlink. For example

```sh
mv cachedModules nowHere
ln -s nowHere cachedModules
```

Make sure you can still start the game and that it is able to load modules.
